pkg/router/template/certmanager.go

```diff
@@ -12,10 +12,25 @@ import (
 	routeapi "github.com/openshift/origin/pkg/route/apis/route"
 )
 
+// certificateFile represents a certificate file.
+type certificateFile struct {
+	CertDir  string
+	ID       string
+	Contents []byte
+}
+
+// certificateFileTag generates a certificate file tag/name. This is used to
+// index into the map of deleted certificates.
+func (cf certificateFile) Tag() string {
```

Comment:
The first word in the comment should match the function name (just `Tag`).

Reply:
Good catch.